src/task/NoOpTask.ts

```diff
@@ -11,7 +11,7 @@ export default class NoOpTask implements Task {
   }
 
   run(): Promise<void> {
-    return;
+    return Promise.resolve();
   }
```

Comment:
This fix is the result of a disappointingly large amount of wasted time wondering why my tests were broken!